<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>220</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>220</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget, QGroupBox{
background: rgb(30, 30, 46);
color: rgb(205, 214, 244);
border: none;
font-family: &quot;Hack Nerd Font Mono&quot;, &quot;Monaco&quot;;
}

#PSFrame, #RelayFrame {
border-width: 2px;
border-radius: 8px;
border-color: rgb(69, 71, 90);
border-style: inset;
}

QComboBox {
background: rgb(186, 194, 222);
border: none;
border-radius: 3px;
font-family: &quot;Hack Nerd Font Mono&quot;, &quot;Monaco&quot;;
}

QComboBox QAbstractItemView {
background: rgb(166, 173, 200);
border-radius: 6px;
padding-top: 3px;
padding-left: 4px;
}

QComboBox::drop-down {
subcontrol-origin: padding;
subcontrol-position: top right;
width: 15px;
border-left-width: 1px;
border-left-color: darkgray;
border-left-style: solid;
border-top-right-radius: 3px;
border-bottom-right-radius: 3px;
}

QComboBox::down-arrow{ 
width: 0;
height: 0;
border-style: solid;
border-right: 3px solid rgb(186, 194, 222);
border-left: 3px solid rgb(186, 194, 222);
border-top: 5px solid rgb(69, 71, 90);
border-bottom: 0;
}

QDoubleSpinBox {
background: rgb(180, 190, 254);
border-radius: 3px;
font-family: &quot;Hack Nerd Font Mono&quot;, &quot;Monaco&quot;;
}

QDoubleSpinBox::up-button, QDoubleSpinBox::down-button {
border: none;
border-left-width: 1px;
border-left-color: darkgray;
border-left-style: solid;
}

QDoubleSpinBox::up-arrow, QDoubleSpinBox::down-arrow {
  width: 0;
  height: 0;
  border-style: solid;
  border-right: 3px solid rgb(186, 194, 222);
  border-left: 3px solid rgb(186, 194, 222);
  border-bottom: 5px solid rgb(69, 71, 90);
  border-top: 0;
}

QDoubleSpinBox::down-arrow {
border-top: 5px solid rgb(69, 71, 90);
border-bottom: 0;
}

QPushButton {
background: rgb(243, 139, 168);
border-width: 2px;
border-radius: 10px;
border-color: rgb(69, 71, 90);
text-align: center;
text-decoration: none;
font-family: &quot;Hack Nerd Font Mono&quot;, &quot;Monaco&quot;;
}

QPushButton:hover {
background-color: rgb(235, 160, 172);
border-style: outset;
}

QPushButton::pressed {
border-style: inset;
}

QPushButton::checked {
  background-color: rgb(166, 227, 161); /* Green */
}

QPushButton::checked:hover {
  background-color: rgb(166, 209, 137); /* Teal */
}

#PSButton::checked {
border-style: inset;
}

#PSButton::checked:hover {
border-style: inset;
}

#BatteryButton, #IGButton{
border: none;
border-radius: 10px;
}

#Close {
background: transparent;
}

#Minimise {
background: transparent;
}

#Theme {
background: rgb(239, 241, 245);
}

#PSButton {
border-radius: 64%;
}</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QGroupBox" name="PSFrame">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>40</y>
      <width>371</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Power Supply</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QGroupBox" name="PSComPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>190</width>
       <height>41</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>COM Port</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QComboBox" name="PSCom">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>17</y>
        <width>172</width>
        <height>20</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="minimumContentsLength">
       <number>0</number>
      </property>
      <property name="placeholderText">
       <string>Select Power Supply COM port</string>
      </property>
      <property name="duplicatesEnabled">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="PSButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>130</width>
       <height>130</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/power_button.png</normaloff>icons/power_button.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="OnVoltageGroup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>On Voltage</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="QDoubleSpinBox" name="OnVoltage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>16</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="keyboardTracking">
       <bool>true</bool>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="suffix">
       <string>v</string>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>40.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::DefaultStepType</enum>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="OffVoltageGroup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>Off Voltage</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="QDoubleSpinBox" name="OffVoltage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>16</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="suffix">
       <string>v</string>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>40.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::DefaultStepType</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="RelayFrame">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>201</width>
      <height>171</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="title">
     <string notr="true">Relays</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QGroupBox" name="IGRelay">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>IG Relay</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="customComboBox" name="IGCom">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>16</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxVisibleItems">
       <number>8</number>
      </property>
      <property name="maxCount">
       <number>8</number>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
      <property name="placeholderText">
       <string>T15 relay ?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="IGButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>16</y>
        <width>60</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/offswitch.png</normaloff>
        <normalon>icons/onswitch.png</normalon>icons/offswitch.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>65</width>
        <height>20</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="BatteryRelay">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>Battery Relay</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="customComboBox" name="BatteryCom">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>16</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="maxVisibleItems">
       <number>8</number>
      </property>
      <property name="maxCount">
       <number>8</number>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAtBottom</enum>
      </property>
      <property name="placeholderText">
       <string>T30 relay?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BatteryButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>16</y>
        <width>60</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/offswitch.png</normaloff>
        <normalon>icons/onswitch.png</normalon>icons/offswitch.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>65</width>
        <height>20</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="RelayComPort">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>COM Port</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QComboBox" name="RelayCom">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>17</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
      <property name="minimumContentsLength">
       <number>0</number>
      </property>
      <property name="placeholderText">
       <string>Select Power Supply COM port</string>
      </property>
      <property name="duplicatesEnabled">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="TitleFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="author">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Harish Kathalingam (uif51939)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="toolTipDuration">
      <number>-4</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>icons/Dev.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Hack Nerd Font Mono</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(250, 176, 5);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>PSGui</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="logo">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>icons/electric.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ReloadCss">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ReloadCss</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Close">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/close.svg</normaloff>icons/close.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Theme">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/light_mode.png</normaloff>
       <normalon>icons/dark_mode.png</normalon>icons/light_mode.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Minimise">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>10</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/minimise.png</normaloff>icons/minimise.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>customComboBox</class>
   <extends>QComboBox</extends>
   <header>customcombobox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
